ansible/ci-build-collector-full.yml

```diff
@@ -19,7 +19,7 @@
     - name: Build full image
       when:
         - build_full_image
-        - arch != 'arm64'
+        - arch != 'arm64' and arch != 'ppc64le' and arch != 's390x'
```

Comment:
revert before merging

Reply:
Not reverting - will be removed when the midstream is updated with this module version